### ISSUE TITLE
big_map_get fails for first contract on chain

### ISSUE THREAD

**Issue by agtilden** [baking-bad/pytezos#126]
The following simple contract associates the given key with the sender's address: 

```{ parameter string ;
  storage (big_map string address) ;
  code { DUP ; CDR ; SENDER ; SOME ; DIG 2 ; CAR ; UPDATE ; NIL operation ; PAIR } }
```

Using flextesa create a brand new chain and originate this contract. Assume contract id is KT1KGks4mu3ZvcQwydAyPHZUbtr7w1e5aozz for this example.

Use pytezos to invoke with a simple string argument. e.g. "foo". 

```
ci = ptz.contract("KT1KGks4mu3ZvcQwydAyPHZUbtr7w1e5aozz")
ci.call("foo").operation_group.sign().inject()
```

Observe tezos-client is able to retrieve value from the bigmap: 

```tezos-client get big map value for '"foo"' of type string in KT1KGks4mu3ZvcQwydAyPHZUbtr7w1e5aozz
Warning:
  
   The node you are connecting to claims to be running in a
                    Tezos TEST SANDBOX.
      Do NOT use your fundraiser keys on this network.
  You should not see this message if you are not a developer.

"tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb"
```

Attempt to retrieve the same value using pytezos: 

```ci.big_map_get("/foo")                                                                                                                                                                                                     
$schema:
	{ string : $address , ... }  /* big_map */

$address:
	string  /* Base58 encoded `tz` or `KT` address */

{'string': 'tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb'}
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
~/.venv/lib/python3.7/site-packages/pytezos/michelson/converter.py in decode_micheline(val_expr, type_expr, schema, root)
     44     try:
---> 45         return parse_micheline(val_expr, type_expr, schema, root)
     46     except (KeyError, IndexError, TypeError) as e:

~/.venv/lib/python3.7/site-packages/pytezos/michelson/micheline.py in parse_micheline(val_expr, type_expr, schema, bin_root)
    252     for idx in bin_root[1:]:
--> 253         type_expr = type_expr['args'][int(idx)]
    254 

KeyError: 'args'

During handling of the above exception, another exception occurred:

MichelineSchemaError                      Traceback (most recent call last)
<ipython-input-5-641fec236417> in <module>
----> 1 ci.big_map_get("/foo")

~/.venv/lib/python3.7/site-packages/pytezos/tools/docstring.py in __call__(self, *args, **kwargs)
     65 
     66                 def __call__(self, *args, **kwargs):
---> 67                     return method(self.class_instance, *args, **kwargs)
     68 
     69                 def __repr__(self):

~/.venv/lib/python3.7/site-packages/pytezos/michelson/interface.py in big_map_get(self, path, block_id)
    390         else:
    391             value = self.shell.blocks[block_id].context.contracts[self.address].big_map_get.post(query)
--> 392         return self.contract.storage.big_map_decode(value, query.get('big_map_id'))
    393 
    394     def storage(self, block_id='head'):

~/.venv/lib/python3.7/site-packages/pytezos/tools/docstring.py in __call__(self, *args, **kwargs)
     65 
     66                 def __call__(self, *args, **kwargs):
---> 67                     return method(self.class_instance, *args, **kwargs)
     68 
     69                 def __repr__(self):

~/.venv/lib/python3.7/site-packages/pytezos/michelson/contract.py in big_map_decode(self, value, big_map_id)
    227                                 type_expr=self.code,
    228                                 schema=self.schema,
--> 229                                 root=value_root)
    230 
    231     def big_map_diff_decode(self, diff: list) -> dict:

~/.venv/lib/python3.7/site-packages/pytezos/michelson/converter.py in decode_micheline(val_expr, type_expr, schema, root)
     47         print(generate_docstring(schema, 'schema'))
     48         pprint(val_expr, compact=True)
---> 49         raise MichelineSchemaError(f'Failed to decode micheline expression', e.args)
     50 
     51 

MichelineSchemaError: ('Failed to decode micheline expression', ('args',))

```

This appears to only affect the **FIRST** contract originated on chain. Originate another contract using the same michelson code. Observe second time around works like a charm.

**Comment by m-kus** [comment]
That was probably a cache issue, pls try with `2.5.9`.
If the problem still exists, I will need the full python script.

**Comment by agtilden** [comment]
This is still a problem on 2.5.9. Script and steps to reproduce available at https://gist.github.com/agtilden/7fcb002b117d7bc0a1d3a01128a92a79

**Comment by m-kus** [comment]
That was an issue with Big_map_id == 0, should be fixed in `2.5.11`  
Btw you can inject synchronously both operations and contract calls:
```
pytezos.origination(c.script()).autofill().sign().inject(_async=False)
ci.endorse("hello").inject(_async=False)  # added in 2.5.11
```

Here is the doc: https://pytezos.org/operation_groups.html#pytezos.operation.group.OperationGroup.inject  

I have tested on a flextesa instance and it seems to work fine.
Please let me know if it works with your setup.
